modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -401,6 +401,18 @@ CV_EXPORTS_W void imshow(const String& winname, InputArray mat);
  */
 CV_EXPORTS_W void resizeWindow(const String& winname, int width, int height);
 
+/** @brief Resizes window to the specified size
```

Comment:
Good point - it's been a while since I used doxygen. Fixing.
Also, originally, I wanted to make it an inline function calling the one above, but I wasn't sure how that plays with the `CV_EXPORTS_W` macro.